chemprop/cli/fingerprint.py

```diff
@@ -40,10 +45,12 @@ def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
             help="Path to which predictions will be saved. If the file extension is .npz, they will be saved as a npz file, respectively. Otherwise, will save predictions as a CSV. The index of the model will be appended to the filename's stem. By default, predictions will be saved to the same location as '--test-path' with '_fps' appended, i.e., 'PATH/TO/TEST_PATH_fps_0.csv'.",
         )
         parser.add_argument(
+            "--model-paths",
```

Comment:
We don't have this in predict.py yet. I would prefer to add it, but I'm not sure whether we should make `--model-paths` or `--model-path` the priority.

Reply:
True, this PR should probably wait after your PR https://github.com/chemprop/chemprop/pull/919 is merged. What ever is decided in your PR, I will make the same changes accordingly for consistency.

Reply:
I will make some changes related to this flag in my PR.